.readthedocs.yml

```diff
@@ -5,7 +5,7 @@
 version: 2
 
 build:
-  os: ubuntu-22.04
+  os: ubuntu-24.04
```

Comment:
How do you test this?

Reply:
I don't have access to it, @sarahelsaig can you please check? If it's a hassle, we can just omit this change, it's not that important for the docs.

Reply:
I don't know. @agriffard ?

Reply:
I just remembered, and yes, we need Antoine's help here. @BenedekFarkas @sarahelsaig please sign up with your Lombiq addresses here: https://readthedocs.org/dashboard/

Then, Antoine, please add benedek.farkas@lombiq.com, sara.el-saig@lombiq.com, and zoltan.lehoczky@lombiq.com as maintainers.

FYI for OC, there's an OC project there, the admin of which you can access under https://readthedocs.org/projects/orchardcore/. Bence, once you sign up with, I can add you there too.

Once done, I _think_ you can add a new version under https://readthedocs.org/projects/orchardcore/versions/ for the branch and build the docs only for that.

The whole thing should be analogous for OCC too.

Reply:
zoltan.lehoczky@lombiq.com added as maintainer.
benedek.farkas@lombiq.com and sara.el-saig@lombiq.com don't currently exist as user on RTD, tell me when I can add them.

Reply:
Ah, there's an invitation e-mail. Accepted it, now it works.

I activated the version and a build is now running.

Reply:
Works: https://commerce.orchardcore.net/en/issue-infra-164/ I now deactivated it.

Nevertheless, Bence, Sári, please still sign up to RTD.

Reply:
I've signed up now.

Reply:
Added you.

Reply:
I've just signed up as well.

Reply:
Added you too.

Reply:
I'm in, thanks!